source/quick_start_guide/index.html.md.erb

```diff
@@ -25,11 +25,20 @@ The following are some examples of base URLs:
 
 <div style="height:1px;font-size:1px;">&nbsp;</div>
 
-### Choose an endpoint  
+### Append the current version of the API
+The current version of the API is **v1**.
+
+You should add the version of the API you want to use to the end of the base URL. This step is optional for version 1 of the API and will become required when version 1 is no longer supported.
```

Comment:
Again not sure if we should use the term "v1" throughout rather than "version 1" or "Version 1".

Perhaps:

> ... This is optional for v1 of the API. When v1 is no longer supported, this will be required.

Reply:
that makes sense - I think it's better to be consistent with what's in the URL